server/subscription/buffer.go

```diff
@@ -0,0 +1,114 @@
+package subscription
```

Comment:
Brainstorm: does it make sense in the future to start to using queue engines to deal with event messaging?
I was thinking if we could use something like https://github.com/adjust/rmq or similar.

Reply:
I think it makes sense to close this PR and to work with some queue engine.

Reply:
ok!